apps/src/applab/gridUtils.js

```diff
@@ -38,15 +37,37 @@ module.exports.scaledDropPoint = function (draggedElement) {
     left: left,
     top: top
   };
-};
+}
+
+/**
+ * Get jQuery object for the element(s) currently being dragged. Will be empty
+ * if no drag is currently underway.
+ * @returns {jQuery}
+ */
+export function getDraggedElement() {
+  return $('.ui-draggable-dragging');
+}
+
+/**
+ * If a drag is underway, returns the target coordinates of the dragged element
+ * if they were dropped right now.  If no drag is underway, returns null.
+ * @returns {TopLeft|null}
+ */
+export function draggedElementDropPoint() {
+  const draggedElement = getDraggedElement();
+  if (!draggedElement.length) {
+    return null;
+  }
+  return module.exports.scaledDropPoint(draggedElement);
```

Comment:
might as well just call `scaledDropPoint()` directly instead of using `module.exports`